tfhe-zk-pok/src/curve_api.rs

```diff
@@ -198,19 +224,32 @@ impl CurveGroupOps<bls12_381::Zp> for bls12_381::G1 {
     fn double(self) -> Self {
         self.double()
     }
+
+    fn normalize(self) -> Self::Affine {
```

Comment:
normalization is the act of getting the affine representation of a projective element ?

Reply:
yes